src/Nest/ElasticClient.cs

```diff
@@ -46,6 +47,28 @@ private CatResponse<TCatRecord> DeserializeCatResponse<TCatRecord>(IApiCallDetai
 			return catResponse;
 		}
 
+		private CatResponse<TCatRecord> DeserializeCatHelpResponse<TCatRecord>(IApiCallDetails response, Stream stream)
+			where TCatRecord : ICatRecord

```

Comment:
does this need to be generic over `TCatRecord`, or can it simply be `CatHelpRecord`?

Reply:
That would involve removing that generic argument from the entire method invocation list, which is a much bigger job...

Reply:
Will replace with `CatHelpRecord`